src/Hooks/Frontend.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+class Frontend implements \IWPML_Frontend_Action, \IWPML_DIC_Action {
+
+    public function add_hooks() {
+        add_action( 'elementor_pro/search_form/after_input', [ $this, 'elementor_add_language_form_field' ] );
+    }
+
+    public function elementor_add_language_form_field() {
```

Comment:
@GeorgeBotsev, for new code (and when it's not forced by an interface), we want to use camelCase. Could you please rename if `addLanguageFormField`?

Note: We don't need to mention `elementor` in the method name, this is implicit.